mm2src/coins/utxo/utxo_builder/utxo_coin_builder.rs

```diff
@@ -256,6 +258,19 @@ pub trait UtxoFieldsWithHardwareWalletBuilder: UtxoCoinBuilderCommonOps {
     }
 
     fn gap_limit(&self) -> u32 { self.activation_params().gap_limit.unwrap_or(DEFAULT_GAP_LIMIT) }
+
+    fn supports_trezor(&self, conf: &UtxoCoinConf) -> bool { conf.trezor_coin.is_some() }
```

Comment:
This is the same code as <https://github.com/KomodoPlatform/atomicDEX-API/pull/1227/files#diff-23f5c94669b4a5a1821afdb607892331c6245d41f68c1156bf19c8c91e0db27aR549> is there a way to have a common function for both?

Reply:
When we add Ledger support, `UtxoFieldsWithHardwareWalletBuilder::supports_trezor` will stay unchanged, but `UtxoCoinBuilderOps::is_hw_coin` will be changed:
```rust
fn is_hw_coin(&self, conf: &UtxoCoinConf) -> bool { conf.trezor_coin.is_some() || conf.ledger_coin.is_some() }
```

I agree with you that it will be nice to have:
```rust
fn is_hw_coin(&self, conf: &UtxoCoinConf) -> bool { self.supports_trezor() || self.supports_ledger() }
```
But `supports_trezor`, `supports_ledger` are associated with the `UtxoFieldsWithHardwareWalletBuilder` trait, and `UtxoCoinBuilderOps` can't inherit `UtxoFieldsWithHardwareWalletBuilder`.